src/targets/axis_2400.ini

```diff
@@ -0,0 +1,45 @@
+# ********************************
+# Transmitter targets
+# ********************************
+[env:AXIS_THOR_2400_TX_via_UART]
+extends = env_common_esp32
+build_flags =
+	${env_common_esp32.build_flags}
+	${common_env_data.build_flags_tx}
+	-include target/AXIS_THOR_2400_TX.h
+	-D TARGET_AXIS_THOR_2400_TX=1
+	-D USER_SETUP_LOADED=1
+	-D CONFIG_DISABLE_HAL_LOCKS=1
+	-include lib/SCREEN/TFT_eSPI_User_Setup.h
+	-D VTABLES_IN_FLASH=1
+	-O2
+src_filter = ${env_common_esp32.src_filter} -<rx_*.cpp>
+lib_deps =
+    ${env_common_esp32.lib_deps}
+    https://github.com/jonesfu/TFT_eSPI.git
```

Comment:
Any displays should be vendor independent.

Reply:
I know, but for TFT_eSPI the software just only work fine on ESP32 not other chips, my PR is not approved at https://github.com/Bodmer/TFT_eSPI/pull/1377.
At now, it is difficult to find simple and good vendor library to support TFT and we keep work on it.

Reply:
I think FOG_Yamato is wanting to use ucglib for the TFT code.

Reply:
In the mean time, this can be changed to `bodmer/TFT_eSPI` If the changes are made to use HSPI rather than software SPI.

Reply:
fixed it at d617ecf